onnxruntime/test/optimizer/qdq_transformer_test.cc

```diff
@@ -1199,6 +1197,76 @@ TEST(QDQTransformerTests, ConvAveragePoolReshape_Int8_Fail) {
   test_case({1, 22, 11, 13, 15}, {30, 22, 5, 3, 3});
 }
 
+template <typename InputType, typename OutputType>
+void QDQTransformerLeakyReluTests() {
+  auto test_case = [&](const std::vector<int64_t>& input_shape) {
+    auto build_test_case = [&](ModelTestBuilder& builder) {
+      auto* input_arg = builder.MakeInput<float>(input_shape, -1.f, 1.f);
```

Comment:
Is int32 a feasible input type that we should add test for it?

Reply:
https://github.com/onnx/onnx/blob/master/docs/Operators.md#leakyrelu